tests/fsharpqa/testenv/bin/.gitignore

```diff
@@ -1,7 +1,8 @@
-FSharp.Compiler.*
-FSharp.Core.*
-HostedCompilerServer.*
-ILComparer.*
-diff.*
-x86
-AMD64
\ No newline at end of file
+/FSharp.Compiler.*
+/FSharp.Core.*
```

Comment:
It will explicitly match at the root of the repository instead of recursively in subdirectories. Seems fine.